.storybook/main.js

```diff
@@ -1,13 +1,12 @@
-import { dirname, join } from 'path';
 /**
  * Copyright Zendesk, Inc.
  *
  * Use of this source code is governed under the Apache License, Version 2.0
  * found at http://www.apache.org/licenses/LICENSE-2.0.
  */
 
-const path = require('path');
-const { readdirSync } = require('fs');
+import path from 'node:path';
```

Comment:
Should this be ..

```suggestion
const { dirname, join } = require('node:path');
```

.. until we're ready to upgrade to ESM? :shrug:

Reply:
`path.join` and `path.dirname` are static methods on `path` as well as named exports, so I think both are valid? I condensed it into one import for simplicity.

- https://nodejs.org/api/path.html#pathdirnamepath
- https://nodejs.org/api/path.html#pathjoinpaths